lambda-function/package.json

```diff
@@ -24,6 +24,7 @@
     "@types/node": "^17.0.13",
     "@types/nodemailer": "^6.4.14",
     "@types/nodemailer-smtp-transport": "^2.7.8",
+    "@types/uuid": "^9.0.8",
```

Comment:
we can use nodejs `crypto` so probably no need to add new package. we are using it for generating population IDs I think already  https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID

Reply:
I see that, updated to use Crypto instead of a new package